vkcs/compute/resource_instance_test.go

```diff
@@ -1500,20 +1501,20 @@ const testAccComputeInstanceNetworkNameToID = `
 {{.BaseImage}}
 {{.BaseFlavor}}
 
-resource "vkcs_networking_network" "network_1" {
-  name = "network_1"
+resource "vkcs_networking_network" "network_acc_test" {
```

Comment:
There is no need to make the resource name and the cloud object name identical.

Reply:
I know, I did this so that all network were names similarly